Resources/doc/first_steps.rst

```diff
@@ -0,0 +1,456 @@
+    {
+        $product = $this->get('doctrine_mongodb')
+            ->getRepository('AcmeStoreBundle:Product')
+            ->find($id);
+
+        if (!$product) {
+            throw $this->createNotFoundException('No product found for id '.$id);
+        }
+
+        // do something, like pass the $product object into a template
+    }
+
+When you query for a particular type of object, you always use what's known
+as its "repository". You can think of a repository as a PHP class whose only
+job is to help you fetch objects of a certain class. You can access the
+repository object for a document class via:
+
+.. code-block:: php
+
+    $repository = $this->get('doctrine_mongodb')
+        ->getManager()
+        ->getRepository('AcmeStoreBundle:Product');
+
+.. note::
+
+    The ``AcmeStoreBundle:Product`` string is a shortcut you can use anywhere
+    in Doctrine instead of the full class name of the document (i.e. ``Acme\StoreBundle\Document\Product``).
+    As long as your document lives under the ``Document`` namespace of your bundle,
+    this will work.
+
+Once you have your repository, you have access to all sorts of helpful methods:
+
+.. code-block:: php
+
+    // query by the identifier (usually "id")
+    $product = $repository->find($id);
+
+    // dynamic method names to find based on a column value
+    $product = $repository->findOneById($id);
+    $product = $repository->findOneByName('foo');
+
+    // find *all* products
+    $products = $repository->findAll();
+
+    // find a group of products based on an arbitrary column value
+    $products = $repository->findByPrice(19.99);
+
+.. note::
+
+    Of course, you can also issue complex queries, which you'll learn more
+    about in the `Querying for Objects`_ section.
+
+You can also take advantage of the useful ``findBy()`` and ``findOneBy()`` methods
+to easily fetch objects based on multiple conditions:
+
+.. code-block:: php
+
+    // query for one product matching be name and price
+    $product = $repository->findOneBy(array('name' => 'foo', 'price' => 19.99));
+
+    // query for all products matching the name, ordered by price
+    $product = $repository->findBy(
+        array('name' => 'foo'),
+        array('price' => 'ASC'),
+    );
+
+Updating an Object
+~~~~~~~~~~~~~~~~~~
+
+Once you've fetched an object from Doctrine, updating it is easy. Suppose
+you have a route that maps a product id to an update action in a controller:
+
+.. code-block:: php
+
+    public function updateAction($id)
+    {
+        $dm = $this->get('doctrine_mongodb')->getManager();
+        $product = $dm->getRepository('AcmeStoreBundle:Product')->find($id);
+
+        if (!$product) {
+            throw $this->createNotFoundException('No product found for id '.$id);
+        }
+
+        $product->setName('New product name!');
+        $dm->flush();
+
+        return $this->redirectToRoute('homepage');
+    }
+
+Updating an object involves just three steps:
```

Comment:
Most of the times, "just" is another of those bad words for newcomers. If you remove it from this phrase, no meaning is lost and it's more friendly.